executor/executor_runner.h

```diff
@@ -621,11 +633,36 @@ class Runner
 				failmsg("unknown host message type", "type=%d", static_cast<int>(raw.msg.type));
 		}
 
+#if GOOS_linux
+		const uint64 kRunLeakCheckEvery = 2 * procs_.size();
```

Comment:
Nit: this logic could be put into a separate method to keep the `Loop` code smaller.